Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -140,4 +140,18 @@ public class BTShopperInsightsClient {
         )
         return error
     }
+
+    // MARK: - Public Methods
```

Comment:
I think we can move this mark to the top of the public methods above and move these methods under `sendVenmoSelectedEvent`

Reply:
`sendVenmoSelectedEvent` is private and I thought we want the new methods public? I moved the mark to the top  with the new methods